docs/src/integrations/observability/blackfire.md

```diff
@@ -1,11 +1,13 @@
 ---
 title: "Blackfire"
 description: |
-  Platform.sh supports [Blackfire.io](https://blackfire.io/). Blackfire is a PHP profiler and automated performance testing tool that can be used in the development Integration, Staging, and Production environments.
+  Platform.sh recommends [Blackfire.io](https://blackfire.io/). Blackfire is a full continuous observability solution (monitoring, profiling, and automated performance testing tool). It can be used in the development Integration, Staging, and Production environments. Blackfire supports PHP and Python.
```

Comment:
```suggestion
  Platform.sh recommends [Blackfire.io](https://blackfire.io/). Blackfire is a full continuous observability solution (monitoring, profiling, and automated performance testing tool). It can be used on Integration, Staging, and Production environments. Blackfire supports PHP and Python.
```